docs/stack/_index.md

```diff
@@ -30,6 +30,10 @@ Redis Stack was created to allow developers to build real-time applications with
 
 Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles five Redis modules: [RedisJSON](/docs/stack/json), [RediSearch](/docs/stack/search), [RedisGraph](/docs/stack/graph), [RedisTimeSeries](/docs/stack/timeseries), and [RedisBloom](/docs/stack/bloom).
 
+### Use cases
+
+Explore [use cases](/docs/stack/use-cases/) of the applied modern data models and processing engines provided by Redis Stack across industries and applications.
```

Comment:
Note that all these modules are generic and we list here only representative use cases.